src/main/java/com/zendesk/maxwell/schema/columndef/ColumnDef.java

```diff
@@ -26,6 +35,16 @@ public Object asJSON(Object value) {
 		return value;
 	}
 
+	public static ColumnDef build(String name, String charset, String type, int pos, boolean signed, String enumValues[], Long columnLength) {
```

Comment:
no need for a polymorphic `build` function -- just pass `null` for `columnLength` in the cases where you don't have it.